CHANGELOG.md

```diff
@@ -1,5 +1,377 @@
 # Changelog
 
+## [1.12.0] (2024-04-09)
+
+_Feature freeze: 2024-03-26_
+
```

Comment:
what's the point of this line?

Reply:
The generator script automatically includes the milestone description. And I usually put the freeze date down.
But I suppose that's not really useful after the freeze period has started, so we can just remove it.